.gitignore

```diff
@@ -34,7 +34,7 @@ docs/src/tutorials/
 # committed for packages, but should be committed for applications that require a static
 # environment.
 
-
+*.DS_Store
```

Comment:
This is a mac specifc thing, and should go in your own excludes file (typically `$HOME/.config/git/ignore`)